### ISSUE TITLE
Consistency between slots name: heading (alert) and header (accordion)

### ISSUE THREAD

**Issue by imagoiq** [swisspost/design-system#2590]
It would be great to have the same slot naming between components when the usage is the same.

Currently, [`post-alert` component has a slot named `heading`](https://next.design-system.post.ch/?path=/docs/components-alert--docs) while [`post-accordion-item` has a slot named `header`.](https://next.design-system.post.ch/?path=/docs/components-accordion--docs)

**Comment by gfellerph** [comment]
- [ ] depends on #2571

**Comment by gfellerph** [comment]
Closing as won't fix
